launch_env.sh

```diff
@@ -6,6 +6,7 @@ export NUMEXPR_NUM_THREADS=1
 export OPENBLAS_NUM_THREADS=1
 export VECLIB_MAXIMUM_THREADS=1
 
+# Remove this now?
```

Comment:
We probably want to remove this, however this forces an AGNOS update for everyone. If we merge it at the same time when we actually do an AGNOS update this should be fine.